frontend/src/metabase/query_builder/components/filters/modals/InlineOperatorSelector/InlineOperatorSelector.styled.ts

```diff
@@ -4,24 +4,31 @@ import { color, alpha, darken, lighten } from "metabase/lib/colors";
 
 export const InlineOperatorContainer = styled.div`
   font-weight: bold;
-  margin: ${space(1)} 0;
+  font-size: 1rem;
+  margin-bottom: 0.875rem;
+  display: inline-flex;
+  align-items: flex-start;
 `;
 
 export const FieldTitle = styled.span`
   color: ${color("text-dark")};
+  margin-right: ${space(1)};
 `;
 
-export const TableTitle = styled.div`
+export const TableTitle = styled.span`
   color: ${color("text-dark")};
-  margin-bottom: ${space(2)};
   span.light {
```

Comment:
nit: Can it be refactored to a regular styled component without the `.light` class?